src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -1500,5 +1500,200 @@ public static void Main()
                 Diagnostic(ErrorCode.ERR_EOFExpected, "}").WithLocation(11, 1)
             );
         }
+
+        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

Can you please add a test demonstrating correct implementation of the following spec sentence when there is an inherited property:
"If no concrete (i.e. non-abstract) property with a get accessor and with this name and type is explicitly declared or inherited, it is produced by the compiler as follows"
That is, please check that we do *not* generate a property when one is inherited. Please test all access levels (including `private`), and when there is no `set` or `init` accessor.

Also please check that the generated `Equals` method in this case does not include the inherited member (also required by the spec).

Reply:
Thanks. I've added tests for those cases, testing for the current behavior. I'll follow up with separate commits if implementation changes are needed.

---
In reply to: [431363317](https://github.com/dotnet/roslyn/pull/44595#discussion_r431363317) [](ancestors = 431363317)